ui/src/app/facility/schedule/exceptions.component.ts

```diff
@@ -58,13 +58,29 @@ export class ExceptionListComponent {
     @Input() hideTitle = false;
     @Input() hideInfo = true;
     @Input() filter: (exception: ExceptionWorkingHours) => boolean;
-    @Output() created = new EventEmitter<ExceptionWorkingHours>();
+    @Output() created = new EventEmitter<ExceptionWorkingHours[]>();
     @Output() removed = new EventEmitter<ExceptionWorkingHours>();
 
+    orderedExceptions: ExceptionWorkingHours[] = [];
+
     constructor(private roomService: RoomService) {
         this.filter = () => true;
     }
 
+    ngOnInit() {
+        this.orderedExceptions = this.exceptions.sort((a, b) => {
+            const aDate = new Date(a.startDate);
```

Comment:
Vai ihan vaan new Date(a.startDate) - new Date(b.startDate) tjsp?
https://stackoverflow.com/questions/46478974/compare-timestamps-in-javascript

Reply:
Päivämääriä ei voi suoraan laskea yhteen mutt jos sen muuntaa millisekunneiksi niin sen voi laskea. .getTime()